XTGETTCAP+capabilities support (#8)
Rockhopper-Technologies/jinxed: doc/faq.rst

Frequently Asked Questions
==========================

Why is Jinxed called Jinxed?
----------------------------------

Jinxed was written to support blessed_ on Windows. We originally wanted to call it Cursed,
to continue the theme, but the name was taken. Jinxed is a synonym for cursed.

Can you add support for a terminal?
-----------------------------------

A select few terminals from the `terminfo(5)`_ database were selected to reduce packaging
size, excluding many historic and unlikely terminal types like ``avatar``.

If we are missing support for a terminal, please let us know its unique ``TERM`` name,
whether it is in the `terminfo(5)`_ database, or provide an equivalent reference or directly

> `create an issue <https://github.com/Rockhopper-Technologies/jinxed/issues>`_ or pull request.

Reviewer: I still think the wording here is hard to follow. Does this match what you intended?

```suggestion
A select few terminals from the `terminfo(5)`_ database were included to reduce packaging
size. Many historic and unlikely terminal types, like ``avatar``, were excluded.

If we are missing support for a terminal, please let us know its unique ``TERM`` name
and whether it is in the `terminfo(5)`_ database, or provide an equivalent reference. To submit,
`create an issue <https://github.com/Rockhopper-Technologies/jinxed/issues>`_
or `pull request <https://github.com/Rockhopper-Technologies/jinxed/pulls>`_.
```

Frequently Asked Questions
==========================

Why is Jinxed called Jinxed?
----------------------------------

> Jinxed was written to support blessed_ on Windows. We originally want to call it Cursed,

Reviewer: Not related to your changes, but since this line is being modified "want" should be "wanted"

to continue the theme, but the name was taken. Jinxed is a synonym for cursed.

Can you add support for a terminal?
-----------------------------------

A select few terminal from `terminfo(5)`_ database were selected, excluding many historic and
unlikely terminal types (like ``avatar``). Rather than distributed a full list of over 1,000
code-generated python files, a small subset is selected.

However, if we're missing support for a terminal please let us know its unique ``TERM`` name,
whether it is in the `terminfo(5)`_ database, or provide an equivalent reference or directly `create

> an issue <https://github.com/Rockhopper-Technologies/jinxed/issues>`_ or pull request.

Reviewer: ```suggestion
A few select terminals from the `terminfo(5)`_ database were included. Rather than distributed a full list of over 1,000 code-generated python files, many historic and unlikely terminal types (like ``avatar``) were excluded.

However, if we're missing support for a terminal please let us know. We will need its unique ``TERM`` name if it is in the `terminfo(5)`_ database or provide an equivalent reference. To submit, `create
an issue <https://github.com/Rockhopper-Technologies/jinxed/issues>`_ or, even better, a pull request.
```

We can also provide stubs for terminal capabilities, like what was done with the (now legacy) win32 
console. Not all terminals can be supported, there a few requirements:

1. The terminal must be detectable programmatically

   We need to be able to identify the terminal in some reasonable way
   and differentiate it from other terminals. This could be through environment variables,

>    (eg. ``TERM``) the :py:mod:`platform` module, ``TTYPE`` telnet negotiation, or some other method.

Reviewer: comma placement

```suggestion
   and differentiate it from other terminals. This could be through environment variables  (e.g. ``TERM``), the :py:mod:`platform` module, ``TTYPE`` telnet negotiation, or some other method.
```
